src/database/models/activities.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
+	const Activity = sequelize.define(
+		'Activity',
+		{
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: DataTypes.INTEGER,
+			},
+			user_id: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			action_id: {
+				allowNull: false,
+				type: DataTypes.INTEGER,
+			},
+			object_id: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			object_type: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			organization_id: {
+				allowNull: false,
+				type: DataTypes.STRING,
+				primaryKey: true,
+			},
+		},
+		{
+			modelName: 'Activity',
+			tableName: 'activities',
+			freezeTableName: true,
+			paranoid: true,
+			indexes: [
+				{
+					fields: ['organization_id', 'user_id'],
```

Comment:
Should we index action id also? In case if we have to search based on any specific action like created / deleted ? @priyanka-TL

Reply:
We cannot delete the activity of any user. Only thing is we can list it, For list as of now org and user id is enough

Reply:
Its like a tracker, we cant delete anything